### DISCUSSION TITLE
What an AMAZING addition to Processing 👏🙌

### DISCUSSION THREAD

**Discussion by stigmollerhansen** [micycle1/PGS#104] [General]
No discussion, just an appraisal, and a huge thank you to everyone involved in making this AMAZING addition to Processing. I can not over-emphasize how valuable and needed this suite of geometry-themed functions has been for me to expand and pursue some of the visual ideas I have otherwise had to abandon because of their complexity. I opened a new world of creative and aesthetic potential when I found this library. I am sharing my excitement and thankfulness to encourage all contributors to keep adding to this library in the future. Chapeaux!

**Comment by micycle1** [comment]
Thanks for kind words man. Glad you find it very helpful.

I'd be keen to see what you make with it.

**Reply by ronenosity** [reply]
Agree with @stigmollerhansen , love this library, thank you for making it!
